sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_container_client.hpp

```diff
@@ -239,9 +239,9 @@ namespace Azure { namespace Storage { namespace Blobs {
      *
      * @param options Optional parameters to execute this function.
      * @param context Context for cancelling long running operations.
-     * @return A GetBlobContainerAccessPolicyResult describing the container's access policy.
+     * @return A BlobContainerAccessPolicy describing the container's access policy.
      */
-    Azure::Response<Models::GetBlobContainerAccessPolicyResult> GetAccessPolicy(
+    Azure::Response<Models::BlobContainerAccessPolicy> GetAccessPolicy(
```

Comment:
This one should drop the `Get` prefix, but keep the `Result` suffix.
```suggestion
    Azure::Response<Models::BlobContainerAccessPolicyResult> GetAccessPolicy(
```

Reply:
Why? I think AccessPolicy is a resource, like `BlobClient::GetProperties` returns `BlobProperties`.

Reply:
Following this guideline, `AccessPolicy` is different from `GetProperties` and should be a struct wrapper with `Result` suffix. 

> If a request to the service only returns response   headers (and no response body), such as a HEAD request, OR only contains a  response body and “no meaningful response headers”. This becomes the T in the   Response<T>.

From today's discussion return the 4 headers (including etag, lastmodified, etc.):

Reply:
Then I'm gonna name the return type `GetBlobContainerAccessPolicyResult`, and maybe `BlobContainerAccessPolicyResult` if Ted signs off on your proposal.